Trippy/Utils/TSPUtils.m

```diff
@@ -0,0 +1,95 @@
+//
+//  TSPUtils.m
+//  Trippy
+//
+//  Created by Catherine Lu on 7/19/22.
+//
+
+#import "TSPUtils.h"
+#import "Itinerary.h"
+
+@implementation TSPUtils
+
++ (NSArray *)permutations:(NSArray *)elements {
```

Comment:
nit-ish: this is probably not the first "private" method like this amongst your code, but I think it'd be better organization to move all your private methods to the bottom of the file, and put a 
```
#pragma mark - Private
```
just above them.